css-four-column-layout/styles.css

```diff
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Barlow', sans-serif;
+}
+
+.container {
+  padding: 0 2rem;
+}
+
+.row1 {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
We generally avoid `.something1`, `.something2`. You can probably just use `text-align` on your heading element.